honeybee_radiance/cli/postprocess.py

```diff
@@ -393,6 +394,53 @@ def annual_metrics(
         sys.exit(0)
 
 
+@post_process.command('annual-daylight-en17037')
+@click.argument(
+    'folder',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, resolve_path=True)
+)
+@click.argument(
+    'schedule',
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, resolve_path=True)
+)
+@click.option(
+    '--grids-filter', '-gf', help='A pattern to filter the grids.', default='*',
+    show_default=True
+)
+@click.option(
+    '--sub_folder', '-sf', help='Optional relative path for subfolder to write output '
+    'metric files.', default='metrics'
+)
+def annual_en17037_metrics(
+    folder, schedule, grids_filter, sub_folder
+):
+    """Compute annual EN 17037 metrics in a folder and write them in a subfolder.
+
+    \b
+    This command generates multiple files for each input grid. Files for target
+    illuminance and minimum illuminance will be calculated for three levels of
+    recommendation: minimum, medium, high.
+
+    \b
+    Args:
+        folder: Results folder. This folder is an output folder of annual
+            daylight recipe. Folder should include grids_info.json and sun-up-hours.txt.
+            The command uses the list in grids_info.json to find the result files for
+            each sensor grid.
+        schedule: An annual schedule for 8760 hours of the year as a list of values. This
+            should be a daylight hours schedule.
```

Comment:
This description is not correct. You are expecting a CSV file, right? Not a list.